src/components/FeeSummary/FeeSummary.vue

```diff
@@ -1,44 +1,51 @@
 <template>
-  <aside id="fee-summary" v-if="showFeeSummary">
-    <affix
-      relative-element-selector=".col-lg-9"
```

Comment:
The affix component needs to know what it's affixing to (CSS selector), which isn't available in this scoped view. So, I moved affix to the parent component.